src/Measurements.jl

```diff
@@ -32,6 +32,10 @@ export Measurement, measurement, ±
 # Re-export from Statistics
 export cor, cov
 
+# Query whether the value is of symbolic type.
+_is_symbolic(::T) where T = false
+_is_symbolic(::DataType) = missing # Call with value, not it's type!
```

Comment:
Why `missing` as opposed to leave the method undefined if it shouldn't be called on datatypes?
Also, there's a typo in the comment
```suggestion
_is_symbolic(::DataType) = missing # Call with value, not its type!
```

Reply:
> Why missing as opposed to leave the method undefined if it shouldn't be called on datatypes?

Because i did in fact happen to accidentally call it on the type (during writing of this PR),
which called `_is_symbolic(::T) where T = false` while i expected `_is_symbolic(Num)` to be called.
It took me more time than i would have liked to understand what is going wrong.
Of course, if preferred, i can drop that, but it's a very obvious bugprone, and easy to avoid.

Reply:
I guess I'm missing how defining this method which returns `missing` helps anything.  Instead of for example throwing a meaningful error message.

Reply:
```
julia> foo() = missing
foo (generic function with 1 method)

julia> if foo() print("hi") else print("no") end
ERROR: TypeError: non-boolean (Missing) used in boolean context
Stacktrace:
 [1] top-level scope
   @ REPL[2]:1


```

Reply:
But maybe the method above should be restricted to `T<:Real`?